.automation/add-to-project-when-labeled

```diff
@@ -38,15 +38,26 @@ except KeyError:
     content_id = event["pull_request"]["id"]
 
 # Fetch the project we want to add to or remove from
-project = [p for p in repo.get_projects() if p.name == LABEL_PROJECTS[label_name]][1]
+try:
+    project = [p for p in repo.get_projects() if p.name == LABEL_PROJECTS[label_name]][0]
+except KeyError:
+    print("Label is not actionable.")
+    sys.exit(0)
 
 # If the label was added, we want to add to the project. If removed, we want to
 # remove from the project.
 if action == "labeled":
     project_column = project.get_columns()[1]
-    project_column.create_card(content_id=content_id, content_type=content_type)
+    try:
+        project_column.create_card(content_id=content_id, content_type=content_type)
+    except github.GithubException as e:
+        if e.data["errors"][0]["message"] == "Project already has the associated issue":
```

Comment:
I don't know if the message that Github returns changes for pull requests (and we won't encounter it in any case at the moment since we can't run on PRs). Do we maybe want to change this to check if the error message starts with `"Project already has the associated"`?

Reply:
Is there an error number we can check instead?

Reply:
Matching on strings can be problematic; particularly when translations get involved.

Reply:
We can match on the the error code (`422`). I'll make that change. Again, no idea if that changes for PRs.